core/src/main/kotlin/api/RunsRoute.kt

```diff
@@ -161,7 +161,7 @@ fun Route.runs() = route("runs/{runId}") {
             call.forRun(ortRunRepository) { ortRun ->
                 requirePermission(RepositoryPermission.READ_ORT_RUNS.roleName(ortRun.repositoryId))
 
-                val pagingOptions = call.pagingOptions(SortProperty("external_id", SortDirection.ASCENDING))
```

Comment:
I wonder whether this should be moved to a preceding fixup commit as it seems to have been wrong in any case, because `RuleViolationsTable` has no `external_id` property. Any opinion @mnonnenmacher?

Reply:
The code shows that the endpoint was copy-pasted from violations, as I just noticed apiDocs is also wrong: it refers to vulnerabilities in endpoint text.

Reply:
Yeah, so that's a good indication that this copy & paste mistake should be fixed separately.

Reply:
But fixing the default sort property requires making another one sortable, so that's an argument for doing it in the same commit.